fig.yml

```diff
@@ -0,0 +1,18 @@
+galera1:
+  build: .
+  command: --wsrep-cluster-address=gcomm://
```

Comment:
But what is the trick behind? Does it not need a cluster address because communication happens just from leader to follower?